src/main/java/domain/Money.java

```diff
@@ -0,0 +1,36 @@
+package domain;
+
+import error.AppException;
+import error.ErrorMessage;
+
+public class Money {
+
+    private static final int UNIT = 1_000;
+    private final int amount;
+
+    public Money(final int amount) {
+        validateRange(amount);
+        validateAmount(amount);
+        this.amount = amount;
+    }
+
+    public int getCount() {
+        return amount / UNIT;
+    }
+
+    private void validateAmount(final int amount) {
+        if (amount % UNIT != 0) {
+            throw new AppException(ErrorMessage.INVALID_MONEY_UNIT);
+        }
+    }
+
+    private void validateRange(final int amount) {
+        if (amount < UNIT) {
+            throw new AppException(ErrorMessage.INVALID_MONEY_RANGE);
+        }
+    }
```

Comment:
약간 사소한 부분일 수 있지만
메서드 호출하는 순서에 따라 배치하면 가독성을 더 높일 수 있어요 😃 
대다수 사람은 위에서 아래로 순서대로 읽는것에 편안함을 느끼니까요!

```suggestion
    private void validateRange(final int amount) {
        if (amount < UNIT) {
            throw new AppException(ErrorMessage.INVALID_MONEY_RANGE);
        }
    }
    
    private void validateAmount(final int amount) {
        if (amount % UNIT != 0) {
            throw new AppException(ErrorMessage.INVALID_MONEY_UNIT);
        }
    }
```

Reply:
validate 함수를 추가하면서 순서까지는 생각 못했던 것 같아요.
다음부터는 이런 부분도 신경 쓰면서 코드 작성하겠습니다!